addons/website_livechat/static/tests/new/thread_tests.js

```diff
@@ -100,7 +100,6 @@ QUnit.test("Livechat with logged visitor should show visitor banner", async func
     const { openDiscuss } = await start();
     await openDiscuss(channelId);
     assert.containsOnce(target, ".o-mail-visitor-banner");
-    assert.containsOnce(target, ".o-mail-visitor-banner:contains(Partner Visitor)");
```

Comment:
the goal of the test was to cover the partner name was used if the visitor was linked to a partner, this change makes this test pretty useless in current state